### PR TITLE
NMS-12733: set system properties for enabling TLSv1.1, TLSv1.2, and TLSv1.3 for …

### PR DESCRIPTION
**Pull request by swachter**
…clients on startup

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-12733

### CODE REVIEW

File: features/poller/monitors/core/src/test/java/org/opennms/netmgt/poller/monitors/TlsTest.java [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+ * It implements a server that can support different TLS protocols.
+ * <p>
+ * It implements a couple of test methods that check if a client can connect using a certain TLS protocol.
+ * <p>
+ * Some of the security related code is taken from the example code of the (great) book
+ * "Java Cryptography: Tools and Techniques"
+ * (cf. https://www.bouncycastle.org/java-crypto-tools-src.zip)
+ */
+@Ignore
+public class TlsTest {
```
Comment (fooker) [inline features/poller/monitors/core/src/test/java/org/opennms/netmgt/poller/monitors/TlsTest.java:93]: Maybe rename this to `TlsExperiment` and removing the `@Ignore` to make more clear, that this is not a unit test.

**Review by fooker — CHANGES_REQUESTED** [review]
Beside the renaming of the experiment code, this is perfectly fine.

File: features/poller/monitors/core/src/test/java/org/opennms/netmgt/poller/monitors/TlsTest.java [hunk 2]
```diff
@@ -0,0 +1,517 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2008-2014 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2014 The OpenNMS Group, Inc.
```
Comment (christianpape) [inline features/poller/monitors/core/src/test/java/org/opennms/netmgt/poller/monitors/TlsTest.java:5]:
```suggestion
 * Copyright (C) 2020 The OpenNMS Group, Inc.
 * OpenNMS(R) is Copyright (C) 1999-2020 The OpenNMS Group, Inc.
```

**Review by christianpape — CHANGES_REQUESTED** [review]
Looks good. Only the header needs to be fixed.

**Review by christianpape — APPROVED** [review]
Ok

**Review by fooker — APPROVED** [review]

**Pull request merged**